processor/transformprocessor/internal/traces/traces.go

```diff
@@ -532,3 +548,25 @@ func setAttr(attrs pcommon.Map, mapKey string, val interface{}) {
 		// TODO(anuraaga): Support set of map type.
 	}
 }
+
+func getTraceState(traceState ptrace.TraceState, mapKey string) interface{} {
```

Comment:
I think the logic is small enough and specific enough to the accessor that this can be moved directly into the Getter instead of its own function.